src/Database/Database.php

```diff
@@ -13,4 +15,6 @@ public function getLastInsertId(): Id;
     public function fetch(Query $query): array;
 
     public function fetchFirst(Query $query): ?array;
+
+    public function transaction(): TransactionManager;
```

Comment:
idk about this name, but I think this would be a nice helper to have

Reply:
I made a comment about it somewhere further down below :)